src/Aspire.Dashboard/Components/Controls/ResourceSelectOptionTemplate.razor

```diff
@@ -6,14 +6,14 @@
 @inject IStringLocalizer<ControlsStrings> Loc
 
 <FluentStack Orientation="Orientation.Horizontal" HorizontalGap="3">
-    @if (ViewModel.Id?.Type is OtlpApplicationType.ReplicaSet)
+    @if (ViewModel.Id is { } typeDetails && typeDetails.Type is OtlpApplicationType.ResourceGrouping)
```

Comment:
Can be a little more concise by matching with a property expression, in this case on the `Type` property:

```suggestion
    @if (ViewModel.Id is { Type: OtlpApplicationType.ResourceGrouping })
```

Reply:
done